cmd/linux.go

```diff
@@ -166,9 +169,9 @@ func RpmInstalled(packageName string) (bool, error) {
 
 // RequestSupportForDistribution promts the user to submit a request to support their
 // currently unsupported distribution
-func RequestSupportForDistribution(distroInfo *DistroInfo, libraryName string) error {
+func RequestSupportForDistribution(distroInfo *DistroInfo) error {
 	var logger = NewLogger()
-	defaultError := fmt.Errorf("unable to check libraries on distribution '%s'. Please ensure that the '%s' equivalent is installed", distroInfo.Name, libraryName)
+	defaultError := fmt.Errorf("unable to check libraries on distribution '%s'.", distroInfo.Name)
```

Comment:
this change makes sense. Seems redundant here to give back the message "Please ensure that the '%s' equivalent is installed"